release-manager-scripts/bom-line-test.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+HERE="$(cd -P "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
```

Comment:
This allows to call the scripts from anywhere and still being able to call colocated scripts.